satpy/etc/readers/seviri_l2_bufr.yaml

```diff
@@ -52,7 +52,7 @@ datasets:
   latitude:
     name: latitude
     key: 'latitude'
-    resolution: [48000,9000]
+    resolution: [48006.450653072,9001.209497451]
```

Comment:
We typically don't use floats for the `resolution` as the user will need to be able to request this resolution and floating point comparison in general is prone to unexpected matching. Do these need to be floats?

Reply:
No, for this reader an integer would be sufficient. I would then probable use the previous implementation with 48000/9000  rather than the more accurate 48006/9001, since the information is not really used and the common user is more familiar with SEVIRI having a resolution of 3000m, rather than 3000.4m, ok?

Fyi: for the `seviri_l2_grib` reader a more accurate resolution is needed, since it's used to compute the area extent, hence I used the float precision here as well.

Reply:
@mraspaud do you have any feeling on this? I'm worried about floating point numbers in the `DataID['resolution']` field, but maybe it isn't that big of a deal?

Reply:
@mraspaud ^

Reply:
seviri has floating point resolutions eg `3000.403165817`, seems to work fine...

Reply:
yeah and the resolution used here is simply multiples (3 or 16) of that value